packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -116,6 +116,15 @@ export const prodParasKusama: EndpointOption[] = [
       Equilibrium: 'wss://node.genshiro.io'
     }
   },
+  {
+    info: 'gm',
```

Comment:
```suggestion
    info: 'gm',
    isUnreachable: true,
```

The endpoint doesn't seem reachable. (Or at least doesn't connect for me)